core/spec/models/spree/order/payment_spec.rb

```diff
@@ -142,20 +136,6 @@ module Spree
       end
     end
 
-    context 'checking if order is paid' do
```

Comment:
@damianlegawiec We have a test for `Spree::Order#paid?` in the order model specs